src/Doctrine/ORM/PixelEventRepository.php

```diff
@@ -79,6 +94,21 @@ protected static function applyDelay(QueryBuilder $qb, int $delay): void
         }
     }
 
+    public function resetFailedByPixel(PixelInterface $pixel): void
+    {
+        $this->createQueryBuilder('o')
+            ->update()
+            ->set('o.state', ':initialState')
+            ->setParameter('initialState', PixelEventInterface::STATE_PENDING, Types::STRING)
+            ->andWhere('o.pixel = :pixel')
+            ->setParameter('pixel', $pixel)
+            ->andWhere('o.state = :state')
+            ->setParameter('state', PixelEventInterface::STATE_FAILED, Types::STRING)
```

Comment:
```suggestion
            ->setParameter('state', PixelEventInterface::STATE_FAILED)
```

Shouldn't we just do this to avoid any future errors? (And then add it to Psalm config)

Reply:
I've added type not because psalm is concerning.

Reply:
Okay